setup.py

```diff
@@ -25,33 +25,31 @@
     long_description = fh.read()
 
 dependencies = [
-    # Dependency corrections from our requirements
-    "attrs==23.1.0",
-    "grpcio==1.53.2",
-    "lazy-object-proxy==1.9.0",
-    "marshmallow==3.19.0",
-    # Core dependencies
-    "fsspec>=2022.8.2",
-    "google-api-python-client==2.91.0",
+    "attrs==24.2.0",
+    "Flask==3.0.3",
```

Comment:
Some versions of airflow such as 2.9.1 depend on flask<2.3 and >=2.2. Can we loosen the Flask version to Flask>=2.2?

Reply:
Changes made here -> https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/1264/commits/434032682d7b0fd32e1353d5041da0b14151388f